### PR TITLE
[FSDE-5245] Implemented Chi version based conditional rendering of Data Table HTML

### PR DESCRIPTION
**Pull request by janibolkvadze**
@dani-cl-madrid @SergioQCL @jllr 

https://ctl.atlassian.net/browse/FSDE-5245

### CODE REVIEW

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-181/1/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://b63b8afdcdc10995d359805692757e98b7448294-chi-pr.enterprise.ctl.io

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-181/2/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-181/3/. :x:

File: src/chi-vue/src/utils/utils.ts [hunk 1]
```diff
@@ -64,3 +64,17 @@ export function copyArrayOfObjects(originArray: any[]) {
...
+
+  if (chiCssLink) {
+    const hrefAttribute = chiCssLink.getAttribute('href');
+
+    if (hrefAttribute) {
+      const version = hrefAttribute.match(/\d+\.\d+\.\d+/);
+
+      return version ? version[0] : null;
+    }
+  }
```
Comment (dani-cl-madrid) [inline src/chi-vue/src/utils/utils.ts:79]: There is no fallback return in the case Chi is not found or the version is not setup as expected. Please, include.

**Review by dani-cl-madrid — CHANGES_REQUESTED** [review]

File: src/chi-vue/src/utils/utils.ts (hunk 1 above)
Comment (janibolkvadze) [reply src/chi-vue/src/utils/utils.ts]: Sounds good, addressed, thanks.

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-181/5/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://fe2b6e8c2664274199bf13eb7ffe4ac7c25768e1-chi-pr.enterprise.ctl.io

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-181/6/. :x:

**Review by aryaroudi — APPROVED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-181/7/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://a089f79cff05661b3b6370636eda68ec5f119f0c-chi-pr.enterprise.ctl.io

**Review by dani-cl-madrid — APPROVED** [review]

**Review by SergioQCL — APPROVED** [review]
ACK

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-181/8/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://1da27894b17daf447edf762fc045f94f1877b87f-chi-pr.enterprise.ctl.io

File: src/chi-vue/src/components/data-table-filters/DataTableFilters.tsx [hunk 2]
```diff
@@ -106,7 +110,7 @@ export default class DataTableFilters extends Vue {
           ${SELECT_CLASSES.SELECT}
           ${this.portal && PORTAL_CLASS}
```
Comment (jllr) [inline src/chi-vue/src/components/data-table-filters/DataTableFilters.tsx:111]: this class must be in v. 4.4.0

File: src/chi-vue/src/components/data-table-filters/DataTableFilters.tsx [hunk 3]
```diff
@@ -363,12 +374,29 @@ export default class DataTableFilters extends Vue {
...
+                }
+                `}>
                 Apply
               </button>
               <button
                 onClick={() => this.toggleDrawer()}
-                class={`${BUTTON_CLASSES.BUTTON} ${BUTTON_CLASSES.PRIMARY} ${BUTTON_CLASSES.SIZES.LG} ${BUTTON_CLASSES.OUTLINE} ${UTILITY_CLASSES.PADDING.X[4]} ${UTILITY_CLASSES.MARGIN.LEFT[2]} -bg--white -uppercase`}>
+                class={`
+                ${BUTTON_CLASSES.BUTTON}
+                ${BUTTON_CLASSES.OUTLINE}
```
Comment (jllr) [inline src/chi-vue/src/components/data-table-filters/DataTableFilters.tsx:392]: this class must be moved for v.4.4.0

File: src/chi-vue/src/components/data-table-views/DataTableViews.tsx [hunk 4]
```diff
@@ -42,7 +48,7 @@ export default class DataTableViews extends Vue {
           <div class={`${FORM_CLASSES.FORM_ITEM}`}>
             <select
               aria-label={`Select a View`}
-              class={`${SELECT_CLASSES.SELECT} ${GENERIC_SIZE_CLASSES.LG}`}
+              class={`${SELECT_CLASSES.SELECT} ${this._chiMajorVersion === 4 ? GENERIC_SIZE_CLASSES.LG : ''}`}
```
Comment (jllr) [inline src/chi-vue/src/components/data-table-views/DataTableViews.tsx:51]: please add -portal class for version 4.4.0

**Review by jllr — CHANGES_REQUESTED** [review]

**Comment by jllr** [comment]
@janibolkvadze also needs to be added -lg class in search input for version 4.4.0

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-181/9/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-181/10/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-181/11/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://a2d77b562b18616592284fdb4e9dc48f0531b6e4-chi-pr.enterprise.ctl.io

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://ae9f7688a1ed2f7cb2f2b8f0722ee31ceab6696d-chi-pr.enterprise.ctl.io

**Review by jllr — APPROVED** [review]

**Pull request merged**